my-app/src/Card.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './App.css';
+
+function Card(props){
+    const [userProfilePicture, setProfilePicture] = React.useState('Loading...')
+    const [userFirstName, setFirstName] = React.useState('Loading...')
+    const [userLastName, setLastName] = React.useState('Loading...')
+    const [userEmail, setEmail] = React.useState('Loading...')
+
+    React.useEffect(() => {
+      async function getUserData(userId) {
```

Comment:
👍 to making a named function for this fetching. I think this works well too. You could also define this outside the component entirely and then use a `then` callback to do the setting of state, but either way works!